code/modules/assembly/signaler.dm

```diff
@@ -241,6 +241,15 @@
 	icon_state = "vortex core"
 	anomaly_type = /obj/effect/anomaly/bhole
 
+/obj/item/assembly/signaler/anomaly/activate()
+	. = ..()
+	var/obj/effect/anomaly/anomaly_path = anomaly_type
+	var/newAnomaly
+	newAnomaly = new anomaly_path(get_turf(src))
+	if(newAnomaly)
+		message_admins("An anomaly has been created via trigger assembly at [ADMIN_VERBOSEJMP(src)]")
```

Comment:
Preferably this would say who triggered it and who installed the signalling device